package.json

```diff
@@ -45,7 +45,29 @@
         "js_bundles": [
             {
                 "name": "packed",
-                "src": "./src-packed/*.js"
+                "src": [
+                    "./src-packed/*.js",
+                    "./src/common/browser-polyfill.min.js",
+                    "./src/config/config.js",
+                    "./src/common/browserDetector.js",
+                    "./src/common/utils.js",
+                    "./src/common/eventBus.js",
+                    "./src/common/messages.js",
+                    "./src/common/i18nManager.js",
+                    "./src/common/extensioni18nManager.js",
+                    "./src/common/languageManager.js",
+                    "./src/common/storageController.js",
+                    "./src/common/extensionStorageController.js",
+                    "./src/common/environmentAdapter.js",
+                    "./src/common/extensionEnvironmentAdapter.js",
+                    "./src/common/extension-init.js",
+                    "./src/common/tracker.js",
+                    "./src/background/graphemeSplitter.js",
+                    "./src/background/validator.js",
+                    "./src/background/dictionarySync.js",
+                    "./src/background/synonyms.js",
+                    "./src/background/extension-main.js"
```

Comment:
Trying to pack all these due to: https://github.com/jonathanpeppers/inclusive-code-comments/pull/90/files#r810203837